docs/examples/accounts

```diff
@@ -36,6 +36,7 @@
 #    ;stunuser=STUN/TURN/ICE-username
 #    ;stunpass=STUN/TURN/ICE-password
 #    ;stunserver=stun:[user:pass]@host[:port]
+#    ;tcpsrcport=6060
```

Comment:
does this config apply to TCP transport only ?

or TCP, TLS, WS, WSS ?

Reply:
TCP and TLS. Because TLS uses also the TCP code.

WS, WSS would need other changes.